app/templates/events/view.hbs

```diff
@@ -35,22 +35,26 @@
       </div>
     </div>
   </div>
-
   <div class="row" style="padding-top: 15px">
     <div class="sixteen wide column">
-      <div class="ui fluid pointing stackable menu {{unless device.isMobile 'secondary'}}">
-        {{#link-to 'events.view.index' class='item'}}
+      {{#events/dropdown-navigation}}
+        {{#link-to 'events.view.index' class='item vertical-item'}}
           {{t 'Overview'}}
         {{/link-to}}
-        <a href="#" class='item'>{{t 'Tickets'}}</a>
-        <a href="#" class='item'>{{t 'Scheduler'}}</a>
-        <a href="#" class='item'>{{t 'Sessions'}}</a>
-        <a href="#" class='item'>{{t 'Speakers'}}</a>
-        <a href="#" class='item'>{{t 'Export'}}</a>
-      </div>
+        {{#link-to 'events.view.tickets' class='item vertical-item'}}
+          {{t 'Tickets'}}
+        {{/link-to}}
+        <a href="#" class='item vertical-item'>{{t 'Scheduler'}}</a>
+        {{#link-to 'events.view.sessions' class='item vertical-item'}}
```

Comment:
Can you add the `vertical-item` class in the component's `didInsertElement` ? Let the developer just follow normal semantic markup. The component can take care of things it requires.